chat-plugins/info.js

```diff
@@ -148,7 +148,8 @@ exports.commands = {
 		if ((user === targetUser || user.can('alts', targetUser)) && hiddenrooms) {
 			buf += `<br />Hidden rooms: ${hiddenrooms}`;
 		}
-		if ((user === targetUser || user.can('makeroom')) && privaterooms) {
+		const lockedAndStaffView = user.can('alts', targetUser) && targetUser.locked;
```

Comment:
Weird name for this permission.

`staffViewingLocked` is probably slightly clearer just because it isn't a "wtf" of English grammar.